reactioncommerce/reaction
# Main navbar container conversion to React

Resolves #2041 

**Testing**
- Clone branch `joykare-main-navbar-2041`

**_Cart_**

- Click on the `cart button` on the navBar
- Expect checkout details to display on the top half of the page (empty cart  || products checkout)

**_Brand_**
_Test 1:_
- Navigate to the dashboard
- Select `Shop` and drag an image on `Drop file to upload`
- Observe the image displayed on the right of the brand name `REACTION`

_Test 2:_
- Navigate to random pages
- Click on Brand name 
- Expect a redirect to home screen

**_Sign out Dropdown_**

-  Click on different links and expect to have same experience as before

**_Whole  Navbar_**

- The whole nav-bar should work and look the same as before
